portfolio/src/main/webapp/script.js

```diff
@@ -29,3 +29,9 @@ function addRandomFact() {
   const factContainer = document.getElementById('fact-container');
   factContainer.innerText = fact; 
 }
+
```

Comment:
Could you add JSDoc to describe the new function?
https://google.github.io/styleguide/jsguide.html#jsdoc

Reply:
Added a commit which adds JSDoc to describe new function getRandomMessage().